src/solution_transfer/radial_basis_interpolation.C

```diff
@@ -27,6 +27,9 @@
 #include "libmesh/libmesh_logging.h"
 #include "libmesh/eigen_core_support.h"
 
+#ifdef LIBMESH_HAVE_EIGEN
+#include <Eigen/Dense>
+#endif
 
```

Comment:
Ah, now I see.  Disregard what I said about the eigen_core_support.h change above.